README.md

```diff
@@ -78,6 +78,11 @@ curl -L https://install.meilisearch.com | sh
 
 If you have the Rust toolchain already installed on your local system, clone the repository and change it to your working directory.
 
+Pre-requitites:
+```
+- Rust 1.40.0
+```
+
```

Comment:
Hello! thank you for your PR! 
meili always compile against latest stable. What do you think about that instead?

```suggestion
If you have the latest stable Rust toolchain already installed on your local system, clone the repository and change it to your working directory.
```

Reply:
@MarinPostma updated~